.github/workflows/scripts/scanner-update-nvd.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+
+import os
+import json
+from datetime import datetime, timedelta
+import logging
+import subprocess
+from urllib import request, parse
+import time
+
+logging.basicConfig(level=logging.INFO)
+
+# Constants
+BASE_URL = "https://services.nvd.nist.gov/rest/json/cves/2.0"
+DIR = "nvd-data"
+API_KEY = os.getenv('API_KEY', None)
+os.makedirs(DIR, exist_ok=True)
+# TODO: ROX-20459 replace scanner v4 test bucket with product bucket
+GCS_BUCKET = "gs://scanner-v4-test"
+
+def compress_and_upload():
+    try:
+        subprocess.run(f"tar -czf {DIR}.tar.gz -C {DIR} .", check=True, shell=True)
+        logging.info(f"Compressed file details: {os.path.getsize(DIR + '.tar.gz')} bytes")
```

Comment:
@jvdm 
I am still using tar.gz since the compressed file is 100MB. Let me know if you think it's still too big of a file.